test/main_dummy_controller_test.cpp

```diff
@@ -0,0 +1,54 @@
+#include "hwlib.hpp"
+#include <dummy_controller_c.hpp>
+
```

Comment:
All of these tests are non-descriptive, what do you check the sliders for? 
And since you have written a dummy controller can't you test this with CATCH?

Reply:
The tests were to check if the interface is working. Catch 2 test are still to be written. this is on the to do list.